packages/agents-manager/src/utils/load-external-providers.ts

```diff
@@ -91,6 +102,10 @@ export async function loadExternalProviders(): Promise< LoadedProviders > {
 			if ( module.useNavigationContinuation ) {
 				mergedNavigationContinuation = module.useNavigationContinuation;
 			}
+			// Collect setup hooks from all providers (multiple plugins can register abilities)
+			if ( module.useAbilitiesSetup ) {
+				setupHooks.push( module.useAbilitiesSetup );
```

Comment:
The `module.useAbilitiesSetup` is just a single function. We could use the same pattern here:

```ts
// Declaration
let mergedAbilitiesSetup: AbilitiesSetupHook | undefined;

// Assignment
if ( module.useAbilitiesSetup ) {
	mergedAbilitiesSetup = module.useAbilitiesSetup;
}

// Usage
useAbilitiesSetup?.();
```